test_user_management.py

```
@@ -0,0 +1,391 @@
+            conn.commit()
+            conn.close()
+        except Exception as e:
+            logging.error(f"Error logging activity: {e}")
+    
+    def backup_database(self, backup_path: str) -> bool:
+        """Create database backup"""
+        try:
+            # SECURITY ISSUE 15: Command injection vulnerability
+            import subprocess
+            command = f"cp {self.db_path} {backup_path}"
+            subprocess.run(command, shell=True, check=True)
```

Ruff (0.15.6)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)